pliers/extractors/audio.py

```diff
@@ -387,3 +495,37 @@ def __init__(self, win_length=384, **kwargs):
 
     def get_feature_names(self):
         return ['tempo_%d' % i for i in range(self.win_length)]
+
+
+    
+class TimeStretchExtractor(LibrosaFeatureExtractor):
```

Comment:
This and `PitchShift` will need to be `Converter` classes rather than `Extractor` classes. They should return a new `AudioStim`.